Mathlib/Geometry/Manifold/Orientable.lean

```diff
@@ -0,0 +1,135 @@
+/-
+Copyright (c) 2024 Rida Hamadani. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rida Hamadani
+-/
+import Mathlib.Geometry.Manifold.SmoothManifoldWithCorners
+
+/-!
+# Orientable Manifolds
+
+This module defines orientable manifolds.
+
+## Main Definitions
+
+- `OrientationPreserving` : a map is orientation-preserving on a given set if the determinant of
+  its Jacobian is strictly positive on that set.
+- `orientationPreservingGroupoid` : the groupoid of partial homeos of `H` which are
```

Comment:
Can you mention the corresponding declarations for orientation-reversing as well?